phys2bids/io.py

```diff
@@ -292,13 +214,67 @@ def read_header_and_channels(filename):
     return header, channel_list
 
 
-def load_txt_ext(filename, chtrig=0):
+def extract_header_items(channel_list, header=[]):
+    """
+    Process items header and channel_list depending on the format (acqKnowledge and labchart).
+
+    Parameters
+    ----------
+    channel_list: list
+        list with channels. For one of the formats it contains header information
+    header: list
+        list with that contains file header
+
+    Returns
+    -------
+    interval: list
```

Comment:
`interval : list of X`

numpydoc puts a space on both sides of the `:`. I'd also have a bit more info about the actual contents of the list. list of str, list of float, etc.